.gitpod.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM gitpod/workspace-base
+
+# Installing Node (LTS)
+# https://github.com/nodesource/distributions/blob/master/README.md#debinstall
+USER gitpod
+RUN curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash -
+RUN sudo apt-get install -y nodejs
+RUN sudo npm install -g typescript yarn pnpm
```

Comment:
I'm not sure if we need `typescript` and `yarn`. The front-end installs `typescript` as one of its dependencies, and I think `yarn` has never been part of `telescope` (but I'd be happy if we started using it).

Reply:
I added `yarn`, as the readme.md in [telescope/src/web/](https://github.com/Seneca-CDOT/telescope/tree/master/src/web#readme) mentions the use of `yarn dev` to run the development server. I'm not sure if we need `typescript` but I added these two while I was going through `telescope` after adding `Node`, `Docker` and `pnpm`, so I could definitely remove them if needed. Please let me know if they are not required.